packages/gatsby-plugin-benchmark-reporting/src/gatsby-node.js

```diff
@@ -79,15 +79,16 @@ class BenchMeta {
 
     const nodejsVersion = process.version
 
-    const gatsbyCliVersion = execToStr(`gatsby --version`)
+    // This assumes the benchmark is started explicitly from `node_modules/.bin/gatsby`, and not a global install
+    const gatsbyCliVersion = require(`gatsby-cli/package.json`).version
```

Comment:
Might be worth a comment saying that this is the same way `gatsby --version` does it.